src/main/java/winston/DoneCommand.java

```diff
@@ -0,0 +1,24 @@
+package winston;
+
+/**
+ *  Represents the done command from the parent abstract class Command.
+ */
+public class DoneCommand extends Command {
+    private final int index;
+    
+    public DoneCommand(TaskList tasklist, int index) {         
```

Comment:
Perhaps a javadoc could be added here for consistency. Currently some classes have javadocs for their constructors and some don't.